### DISCUSSION TITLE
Logout Override (sso logout and dashboard logout)

### DISCUSSION THREAD

**Discussion by Hegabovic** [filamentphp/filament#10761] [Help]
i want to override the logout btn in the dashboard to logout from sso and also log me out from dashboard 

the logic i made logs me out from sso but not from filament dashboard , can you help with that ? 
![Screenshot from 2024-01-11 11-39-33](https://github.com/filamentphp/filament/assets/56083879/0d6ba686-f96b-43cc-aa73-1106ef43c10d)

```
class LogoutResponse implements LogoutResponseContract
{
    public function toResponse($request): RedirectResponse
    {
        $redirectTo = 'http://localhost:9443/logout';

        return redirect($redirectTo);
    }
}

**Comment by Hegabovic** [comment]
@leandrocfe can you provide some help here :D

**Reply by leandrocfe** [reply]
`auth()->logout();` before redirecting?

**Reply by Hegabovic** [reply]
The thing is that i want to fetch the current session id too

**Reply by Hegabovic** [reply]
And it always gives me 419 error

**Reply by Hegabovic** [reply]
Another fast question, it is related with sso too , if i need to replace filament login to when trying to access it redirect to a specific page i assign , is this possible ?

**Comment by mohamedAhmed97** [comment]
i have the same issue with "SSO" too
